.editorconfig

```diff
@@ -3,5 +3,17 @@ root = true
 [*]
 indent_style = tab
 end_of_line = lf
+charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
I don't use/like the "newlines at end of lines add manual linebreaks" feature of markdown, so I would eliminate this

Reply:
fixed